spec/visual/samples/sas

```diff
@@ -0,0 +1,71 @@
+* Example SAS script showing syntax highlighting;
+data population;
+/* 
+The first 10 lines of 2017 population data. Source: World Bank
+https://data.worldbank.org/indicator/sp.pop.totl
+Licence: CC BY-4.0
+
+To demonstrate reading data in-line using datalines.
+*/
+infile datalines dlm = '09'x;
+input
+    country_name: $20.
+    country_code: $3.
+    pop2017;
+datalines;
+Aruba	ABW	105264
+Afghanistan	AFG	35530081
+Angola	AGO	29784193
+Albania	ALB	2873457
+Andorra	AND	76965
+Arab World	ARB	414491886
+United Arab Emirates	ARE	9400145
+Argentina	ARG	44271041
+Armenia	ARM	2930450
+American Samoa	ASM	55641
+;
+run;
+
+* Print first 5 records with over 100k population;
+proc print data=population (obs=10);
+    where pop2017 > 100000;
+run;
+
+* Simulation via a SAS data step;
+data sim;
+    do i = 1 to 100;
+        x1 = rand("Normal");
+        x2 = rand("Binomial", 0.5, 100);
+        output;
+    end;
+run;
+
+* Some analysis of sample data included with SAS;
+proc means data=sashelp.class;
+    class sex;
+    var height weight;
+    output out = mean_by_sex;
+run;
+
+proc freq data=sashelp.cars;
+    tables origin;
+run;
+
+* Fit a regression model;
+proc reg data=sashelp.cars;
+    model msrp = enginesize horsepower mpg_city;
+run;
+
+* Plot Fisher's Iris data;
+proc sgplot data=sashelp.iris;
+    scatter x = SepalLength y = SepalWidth / group = Species;

```

Comment:
Are these supposed to be on the same line?

Reply:
Not particularly. SAS generally ignores newlines, it's looking out for semicolons to delimit statements.